packages/core/src/sass/app/_common/components/account-switcher.scss

```diff
@@ -12,7 +12,7 @@
         position: absolute;
         top: 0;
         right: 0;
-        width: 350px;
+        width: 315px;
```

Comment:
I think it would be better if you replace px with rem (31.5 rem). The older code used px but using rem is the better approach